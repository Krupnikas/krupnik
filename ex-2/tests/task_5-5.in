364853279856327953295623759___________348
28 11
