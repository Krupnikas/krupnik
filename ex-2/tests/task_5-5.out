364853279856327953295623759348
